File: array.rb
# Create the array with 20 random elements from range 0-100.
arr = Array.new(20) { |i| i = rand(0..100)  }	
a = arr.length
puts "Numbers of array's elements is " + a.to_s + "\n" * 2

#Show unsorted array
puts "Array contents:"  
p arr 

# Sort this array
arr.sort!	

puts "\n"

puts "After sort operation, this array contents elements in alphabetical order now:"

#Show this array in alphabetical order
p arr

File: array.php
<p>
<?php
// Create the array with 20 random elements from range 0-100.
$table = array();

for ($i=0; $i < 20; $i++)
		{array_push ($table, rand(0,100));
	};

$count= count($table);

echo "Numbers of array's elements is ".$count.".";
?>
</p>
<p>
<?php
echo "Array contents:";?><br>

<?php

//Print unsorted table.
print_r($table);

//Sort table.
sort($table);
?>
<?php

echo "After sort operation, this array contents elements in alphabetical order now:";?><br>

<?php
//Print sorted array
print_r($table);
unset $table;
?>
</p>